03week/airplane/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        
+        <link rel="stylesheet" type="text/css" href="css/style.css">
+        <link href="https://fonts.googleapis.com/css?family=Lato:100i,300,300i,400" rel="stylesheet">
+        <title>UX Atlantic</title>
+    </head>
+    
+    <body>
+<!-------------------------------------------------------------->
+<!--      Header with nav bar across the top of the page      -->
+<!--------------------------------------------------------------> 
+        <header>
+            <div class="navbar">
+                <a href="www.kayak.com"> <img src="img/logo.png" alt=""></a>
+            </div>
+                <div class="navlinks">
+                    
+                    <div class="registerlogin">
+                    <a><img src="img/login-icon.png"></a>
+                        Register/Login
+                    </div>
+                    
+                    <div class="flightschedule">
+                    <a><img src="img/schedule-icon.png"></a>
+                        Flight Schedule
+                    </div>
+                    
+                    <div class="traveldeals">
+                        <a><img src="img/plane-icon.png"></a>
+                        Travel Deals
+                    </div>
+                    
+                    <div class="home">
+                    <a><img src="img/home-icon.png"></a>
+                        home
+                    </div>
+                    
+                </div>
+            </header>
+<!-------------------------------------------------------------->
+<!-- Below is the main input box on the left side of the page -->
+<!-------------------------------------------------------------->        
+        <div class="aboveleftbox"> 
```

Comment:
These navigational elements did not position correctly when i was looking at it.
I think its because you were using position relative with floats.